### PR TITLE
Cleaning up tox dispatch script

### PR DESCRIPTION
**Pull request by semick-dev**
The core of the script is totally fine. It's just got a lot of cruft built up from features that are unused and unnecessary.  After this PR, the next target for update is the `--filter-type` argument. That argument is totally unnecessary now that we have `pyproject.toml` present in every single package folder!

The yanked code consists of:

- Invocation of the `old style` method of testing that we relied upon when we first _swapped_ to `tox`.
- Options that have gone un-used for the lifetime of the script
 
EDIT: And I absolutely started this entire process with `git mv` of `setup_execute_tests.py`. Sad it's showing it as an entirely new file.

### CODE REVIEW

**Review by mccoyp — APPROVED** [review]
Pretty huge simplification, happy to see it 🙂 Just a tiny nitpick, but otherwise I don't see any problems

File: scripts/devops_tasks/dispatch_tox.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    parser.add_argument(
+        "--tenvparallel",
+        default=False,
+        help=("Flag. Enable tox parallel invocation."),
+        action="store_true",
+    )
+
+    parser.add_argument(
+        "--service",
+        help=("Name of service directory (under sdk/) to test." "Example: --service applicationinsights"),
```
[the remark this answers is not in the document]
Comment (scbedd) [reply scripts/devops_tasks/dispatch_tox.py:74]: Aha! I think this is blank formatting that bled together 👍 

Thanks very much! Mergin.

**Comment by scbedd** [comment]
Going to override and merge. This is an existing core failure.

**Pull request merged**